### PR TITLE
Bug 1160705 - Compare view titles could be prettier and provide more useful information

### PR DESCRIPTION
**Pull request by wlach**
Just putting up some compare perf work for people to look at. Mostly just putting this up so people know there's something in progress. :)

[Review on Reviewable](https://reviewable.io/reviews/mozilla/treeherder-ui/505)

### CODE REVIEW

**Comment by wlach** [comment]
Some pictures of what this looks like:

comparesubtest-newheader
compareall-newheader

**Comment by jmaher** [comment]
shit it

File: webapp/app/js/models/repository.js [hunk 1]
```diff
@@ -130,20 +131,23 @@ treeherder.factory('ThRepositoryModel', [
...
                     }
-
-                    var storedWatched = loadWatchedRepos();
-                    if (_.isArray(storedWatched) && _.contains(storedWatched, name)) {
-                        _.each(storedWatched, function (repo) {
-                            watchRepo(repo);
-                        });
+                    if (options.watchRepos) {
+                        var storedWatched = loadWatchedRepos();
+                        if (_.isArray(storedWatched) && _.contains(storedWatched, name)) {
```
Comment (edmorley) [inline webapp/app/js/models/repository.js]: This needs to be ```_.contains(storedWatched, options.name)```

Comment (wlach) [inline webapp/app/js/models/repository.js]: Touche, fixed.

**Pull request merged**